components/aws/sagemaker/test/integration/resources/simple_mnist_training/simple_mnist_training.py

```diff
@@ -0,0 +1,79 @@
+import kfp
+import json
+import utils
+
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+sagemaker_train_op = components.load_component_from_file("../../train/component.yaml")
+
+CHANNELS = [
+    {
+        "ChannelName": "train",
+        "DataSource": {
+            "S3DataSource": {
+                "S3Uri": "s3://{}/mnist_kmeans_example/data".format(
+                    utils.get_s3_data_bucket()
+                ),
+                "S3DataType": "S3Prefix",
+                "S3DataDistributionType": "FullyReplicated",
+            }
+        },
+        "ContentType": "",
+        "CompressionType": "None",
+        "RecordWrapperType": "None",
+        "InputMode": "File",
+    }
+]
+
+
+@dsl.pipeline(name="SM Training pipeline", description="SageMaker training job test")
```

Comment:
nit: SM is not very clear, just use SageMaker

Reply:
going to change this in next iteration to make the test more generic, this was just intended as sample